lingvo/core/py_utils.py

```diff
@@ -5426,3 +5426,10 @@ def ShardedFilePatternToGlob(file_pattern):
   if shards == '*':
     return f'{path}-?????-of-*'
   return f'{path}-?????-of-{int(shards):05}'
+
+def SequenceToSparseTensor(dense_tensor, padding):
```

Comment:
You can probably revert this if yoru one usage of it is really for an unused variable

Reply:
Oh you are referring to the sparse_gt_ids variable as the dead code in you r previous comment, yes, will fix this one. Will look into pylint.